### PR TITLE
Improve vc+sd-jwt metadata example

### PR DESCRIPTION
**Pull request by selfissued**
This addresses some of the issues raised by @TakahikoKawasaki in #172.

### CODE REVIEW

**Review by Sakurann — APPROVED** [review]
looks ok. please make it clear in the issue, which on Taka's comments have been addressed and which have not, otherwise it will be harder to track later.

**Review by Sakurann — CHANGES_REQUESTED** [review]
per 18013-5, it is `age_over_NN`. please change from `age_65_or_over` to `age_over_65`

**Comment by paulbastian** [comment]
`age_xx_or__over` claims are not in the IANA registry. Should they be prepended with something like an URI to make them public claims?

**Comment by selfissued** [comment]
"Age over 65" means "Age > 65" - which is 66 or greater, in the normal English reading of the phrase.  It's an example.  I'd rather that the example read in a natural way, than have readers think "I know what they meant but that's not what they actually said."

To Paul's point about registering the claims, these are example claims for illustrative purposes only.

**Comment by Sakurann** [comment]
I am afraid this is not an English question. 18013-5 already defined  "age_over_xx" which is used by the governments and other implementations and we should just use it. I am against introducing a new proprietary claim "age_xx_or_over".

**Comment by jogu** [comment]
> "Age over 65" means "Age > 65" - which is 66 or greater, in the normal English reading of the phrase. It's an example. I'd rather that the example read in a natural way, than have readers think "I know what they meant but that's not what they actually said."

I think the interpretation does vary; at least in the United Kingdom it is common to say "you can drink alcohol once you're over 18", which means ">= 18", or perhaps more precisely "> 18.000000".

> To Paul's point about registering the claims, these are example claims for illustrative purposes only.

Unfortunately illustrative examples in specifications have a tendency to get copied and used in real life implementations. If we are not confident in the names or they are contentious etc then I suggest we just remove the "over xx years old" claims from the example here (and perhaps refer people to the actual SD JWT VC spec for more detailed examples).

**Comment by Sakurann** [comment]
I don't understand why there is a controversy around using "age_over_nn" claim which is a relatively established claim in this space.

File: examples/credential_metadata_sd_jwt_vc.json [hunk 1]
```diff
@@ -52,9 +52,9 @@
                     "country": {}
                 },
                 "birthdate": {},
-                "is_over_18": {},
-                "is_over_21": {},
-                "is_over_65": {}
+                "age_18_or_over": {},
+                "age_21_or_over": {},
+                "age_65_or_over": {}
```
Comment (selfissued) [inline examples/credential_metadata_sd_jwt_vc.json:57]:
```suggestion
                "is_over_18": {},
                "is_over_21": {},
                "is_over_65": {}
```

**Comment by selfissued** [comment]
OK, I switched it back to using the ISO claim names.  If it were us creating new misleading claim names, I'd stand by my request for the change, but given we're copying misleading claim names created by ISO, I'll hold my nose and live with it.

**Review by bc-pi — APPROVED** [review]

**Review by paulbastian — APPROVED** [review]

**Review by peppelinux — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]

**Comment by Sakurann** [comment]
@selfissued please make it clear in the original issue which of the taka's issues have been addressed.

**Pull request merged**